include/uapi/sound/sof/tokens.h

```diff
@@ -85,6 +85,7 @@
 #define SOF_TKN_INTEL_DMIC_NUM_PDM_ACTIVE	605
 #define SOF_TKN_INTEL_DMIC_SAMPLE_RATE		608
 #define SOF_TKN_INTEL_DMIC_FIFO_WORD_LENGTH	609
+#define SOF_TKN_INTEL_DMIC_UNMUTE_RAMP_TIME     610
```

Comment:
unit? better to finish with e.g. RAMP_TIME_MS

Reply:
Yes, it's more clear that way.